SettingsView.Droid/SettingsViewRecyclerAdapter.cs

```diff
@@ -60,20 +60,26 @@ void _settingsView_ModelChanged(object sender, EventArgs e)
 
         void OnSectionPropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
         {
+	        if ( !( sender is Section section ) ) return;
```

Comment:
What purpose is this check?

Reply:
It reduces bugs by preventing cast exceptions. This is one of many areas that can be improved in this way.